pom.xml

```diff
@@ -43,6 +43,7 @@
         <module>services/quota_manager</module>
 		<module>services/network_acl_manager</module>
         <module>services/gateway_manager</module>
+        <module>services/pseudo_controller</module>
```

Comment:
@zzxgzgz Rio, is this PR ready to merge? If you plan to get it in before next PR on multiple containers, you could possible do a quick clean up and we can merge.

Reply:
Hi @xieus , I plan to finish the multi port/containers before merging, please give me some time. Thanks